ta.py

```diff
@@ -32,21 +31,19 @@ async def homepage(request):
     return response
 
 async def submit(request):
-    if wants_json(request):
```

Comment:
I think `is_json_client` is much better representative to the case than `wants_json`